cleaning/big_clean.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Robert Beane
+
+# Parameters
+comp=$1 # takes inputed file
+name=$(basename "$comp") # gets full name of file (.tgz)
+noext=$(basename "${comp%.*}") # gets directory name with no extension
```

Comment:
That's some serious voodoo! 😜  I just used `basename <path> <suffix>`, providing `.tgz` as the suffix.